File: livingrimoire start here/LivinGrimoire java/skills java/DiSkillBundle.java
package skills;

import AXJava.AXSkillBundle;
import AXJava.AlgorithmV2;
import LivinGrimoire.DiSkillV2;
import LivinGrimoire.Kokoro;

public class DiSkillBundle extends DiSkillV2 {
    private final AXSkillBundle axSkillBundle = new AXSkillBundle();
    @Override
    public void input(String ear, String skin, String eye) {
        AlgorithmV2 a1 = axSkillBundle.dispenseAlgorithm(ear, skin, eye);
        if(a1 == null){return;}
        this.outAlg = a1.getAlg();
        this.outpAlgPriority = a1.getPriority();
    }

    @Override
    public void setKokoro(Kokoro kokoro) {
        super.setKokoro(kokoro);
        axSkillBundle.setKokoro(kokoro);
    }
    public void addSkill(DiSkillV2 skill){
        axSkillBundle.addSkill(skill);
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/skills/special/DiSkillBundle.kt
package skills.special

import auxiliary_modules.AXSkillBundle
import auxiliary_modules.AlgorithmV2
import livinGrimoire.AbsDictionaryDB
import livinGrimoire.DiSkillV2
import livinGrimoire.Kokoro

class DiSkillBundle : DiSkillV2() {
    private val axSkillBundle: AXSkillBundle = AXSkillBundle()
    override fun input(ear: String, skin: String, eye: String) {
        val a1: AlgorithmV2 = axSkillBundle.dispenseAlgorithm(ear, skin, eye) ?: return
        this.outAlg = a1.getAlg()
        this.outpAlgPriority = a1.priority
    }

    override var kokoro: Kokoro = Kokoro(AbsDictionaryDB())
        set(value) {
            field = value
            kokoro.toHeart.put("dibicameral", "null")
        }

    fun addSkill(skill: DiSkillV2) {
        axSkillBundle.addSkill(skill)
    }
}